elisp/guix-deriviation.el

```diff
@@ -0,0 +1,89 @@
+;;; guix-deriviation.el --- Guix deriviation mode
+
+;; Copyright © 2017 Oleg Pykhalov <go.wigust@gmail.com>
+;; Copyright © 2017 Alex Kost <alezost@gmail.com>
+
+;; This file is part of Emacs-Guix.
+
+;; Emacs-Guix is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; Emacs-Guix is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with Emacs-Guix.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; This package provides minor-mode for prettifying Guix deriviation files.
+
+;;; Code:
+
+(require 'guix-utils)
+
+(defgroup guix-deriviation nil
+  "Prettify Guix deriviation files."
+  :prefix "guix-deriviation-file-"
+  :group 'guix
+  :group 'convenience)
+
+(defcustom guix-deriviation-file-regexp-group 1
```

Comment:
I think this variable is not needed at all.  Just use 0 instead (I mean in `match-beginning`/`match-end`).